mod-entities-links/src/test/java/org/folio/ModEntitiesLinksEurekaTest.java

```diff
@@ -10,7 +10,7 @@
 
 @FolioTest(team = "spitfire", module = "mod-entities-links")
 class ModEntitiesLinksEurekaTest extends TestBaseEureka {
```

Comment:
I guess we should rename this file into ModEntitiesLinksTest and the current ModEntitiesLinksTest which extends  TestBase could be removed